rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -156,6 +156,33 @@ extern "C" RocalTensor ROCAL_API_CALL rocalCropResizeFixed(RocalContext context,
                                                            RocalTensorLayout output_layout = ROCAL_NONE,
                                                            RocalTensorOutputType output_datatype = ROCAL_UINT8);
 
+/*! \brief Fused function which performs fused resize and crop on images with fixed crop.
+ * \ingroup group_rocal_augmentations
+ * \param [in] context Rocal context
+ * \param [in] input Input Rocal tensor
+ * \param [in] dest_height output height
+ * \param [in] dest_width output width
+ * \param [in] crop_h crop width of the tensor
+ * \param [in] crop_w crop height of the tensor
+ * \param [in] crop_pos_x specifies a specific horizontal position for the crop
+ * \param [in] crop_pos_y specifies a specific vertical position for the crop
+ * \param [in] interpolation_type The type of interpolation to be used for resize.
+ * \param [in] is_output is the output tensor part of the graph output
+ * \param [in] output_layout the layout of the output tensor
+ * \param [in] output_datatype the data type of the output tensor
+ * \return RocalTensor
+ */
+extern "C" RocalTensor ROCAL_API_CALL rocalResizeCropFixed(RocalContext context, RocalTensor input,
```

Comment:
What is the difference between rocalResizeCropFixed and rocalCropResizeFixed?

I think rocalCropResizeFixed will be the right name here since we are doing a crop and then reize

Reply:
There is already an API named rocalCropResizeFixed

Reply:
Discussed offline and changed the new API name to rocalROIResize